### ISSUE TITLE
Tabsrmm buttons are missing screen reader labels

### ISSUE THREAD

**Issue by tyrylu** [miranda-ng/miranda-ng#3092]
### Expected behavior
The labels are properly reported

### Actual behavior
The control type is reported as a sole information

### Does it work in stable version?
No.

### Steps to reproduce that behavior
Tested with latest NVDA snapshot and tabsrmm 3.6.2.2 with Miranda NG build 24537

 * Open an conversation window
 * Write at least one character so the buttons are reachable by the tab key (likely a bug in its own right)
 * Press the tab key and observe the missing labels

**Comment by georgehazan** [comment]
@tyrylu 
the problem is that those buttons carry no text at all but tooltips (small yellow windows with text which appear when you hover mouse over a button). I doubt that NVDA can read those tooltips, but there's no place at the toolbar to place buttons with text, it's rather narrow to do that.

**Comment by georgehazan** [comment]
One possible solution is to learn buttons' hotkeys, you can perform any action from keyboard

**Comment by tyrylu** [comment]
You can likely always set the accessibility name for these using something along the lines of https://docs.microsoft.com/en-us/accessibility-tools-docs/items/win32/control_name
No idea whether NVDA will pick them up, however. And when you are at it, setting the proper role should not be such a hard thing either...
